src/impl/peerconnection.cpp

```diff
@@ -736,7 +742,7 @@ shared_ptr<Track> PeerConnection::emplaceTrack(Description::Media description) {
 
 void PeerConnection::iterateTracks(std::function<void(shared_ptr<Track> track)> func) {
 	std::shared_lock lock(mTracksMutex); // read-only
-	for (auto it = mTrackLines.begin(); it != mTrackLines.end(); ++it) {
+	for (auto it = mTracks.begin(); it != mTracks.end(); ++it) {
```

Comment:
What is the point of the change? This idea was that tracks are iterated according to m= line order.